docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -206,8 +206,52 @@ throughout this style guide.
    Include visual of 'Do's and Don'ts!
 
 
-2.0 Dealing with Limitations in Upstream Research
-+++++++++++++++++++++++++++++++++++++++++++++++++
+2.0 Research question
++++++++++++++++++++++
+
+As much as possible, our research should be precise about the variables of interest.
+For example, if we are modeling pregnancy, the variable we are interested in is the
+ability to become pregnant.
+This not only clarifies that we are interested in biological sex and not gender, but is even
+more precise that it is this facet of biological sex that matters,
+not another facet such as chromosomal makeup.
+
+It is frequently not possible to be this precise;
+we should try to at least determine whether our research question is about sex or gender.
+In some cases, even this may not be clear, for example if we are researching an
+association that could be mediated by either sex or gender (or both), making them both
+variables of interest.
+
+2.1 Proxy measures
+------------------
+
+When our input data does not include a variable of interest, we are forced to use
+**proxy measures**.
+A proxy measure is another variable that is highly correlated with the variable of interest,
```

Comment:
When defining your research question, you might have to use proxy measures. A proxy measure is another variable that is highly correlated with the variable of interest, but is not the variable you selected for research. In the pregnancy example...